hyp3_gamma/insar/unwrapping_geocoding.py

```diff
@@ -127,16 +168,21 @@ def unwrapping_geocoding(reference, secondary, step="man", rlooks=10, alooks=2,
 
     execute(f"rascc_mask {ifgname}.adf.cc {mmli} {width} 1 1 0 1 1 0.10 0.20 ", uselogging=True)
 
+    # default reference point in SAR space
+    ref_azlin, ref_rpix = 0, 0
+
+    coords = get_coords(f"{mmli}.par", ref_azlin, ref_rpix, dempar)
```

Comment:
This can just be default values for `get_coords` instead of hard coding them here, can then you can call it like

```python
    coords = get_coords(f"{mmli}.par", in_dem_par=dempar)
```

Reply:
changed